config/config.php

```diff
@@ -6,4 +6,5 @@
         'value' => env('SAVING_QUOTE_EXPIRE_VALUE', 7),
     ],
     'mailable' => null, // Class name of the mailable to send after a quote is saved.
+    'validator' => \Jauntin\SavingQuote\Interfaces\QuoteProgressValidator::class,
```

Comment:
What information does this provide? It's not a class name, so does it actually give Laravel the information it needs to resolve an implementation for this Interface? AFAIK it doesn't - like you'll have to provide the name of the implementation. Assuming I'm correct, this doesn't do anything. `null` plus a good comment like the above makes sense.